python_modules/dagster/dagster/_core/secrets/loader.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABC, abstractmethod
+from typing import Dict, Optional
+
+from dagster._core.instance import MayHaveInstanceWeakref
+
+
+class SecretsLoader(ABC, MayHaveInstanceWeakref):
```

Comment:
is it right to name it secrets ? when we talked irl this is more generic stuff-things-in-to-environment loader. Not sure what the better name is but [1]

Reply:
EnvironmentLoader would also work